qutebrowser/utils/log.py

```diff
@@ -377,151 +360,6 @@ def change_console_formatter(level: int) -> None:
-        # When enabling debugging with QtWebEngine
-        ("Remote debugging server started successfully. Try pointing a "
-         "Chromium-based browser to "),
-        # https://github.com/qutebrowser/qutebrowser/issues/1287
-        "QXcbClipboard: SelectionRequest too old",
-        # https://github.com/qutebrowser/qutebrowser/issues/2071
-        'QXcbWindow: Unhandled client message: ""',
-        # https://codereview.qt-project.org/176831
-        "QObject::disconnect: Unexpected null parameter",
-        # https://bugreports.qt.io/browse/QTBUG-76391
-        "Attribute Qt::AA_ShareOpenGLContexts must be set before "
-        "QCoreApplication is created.",
-        # Qt 6.4 beta 1: https://bugreports.qt.io/browse/QTBUG-104741
-        "GL format 0 is not supported",
-    ]
-    # not using utils.is_mac here, because we can't be sure we can successfully
-    # import the utils module here.
-    if sys.platform == 'darwin':
-        suppressed_msgs += [
-            # https://bugreports.qt.io/browse/QTBUG-47154
-            ('virtual void QSslSocketBackendPrivate::transmit() SSLRead '
-             'failed with: -9805'),
-        ]
-
-    if not msg:
-        msg = "Logged empty message!"
-
-    if any(msg.strip().startswith(pattern) for pattern in suppressed_msgs):
-        level = logging.DEBUG
-    elif context.category == "qt.webenginecontext" and (
-        msg.strip().startswith("GL Type: ") or  # Qt 6.3
-        msg.strip().startswith("GLImplementation:")  # Qt 6.2
-    ):
-        level = logging.DEBUG
-    else:
-        level = qt_to_logging[msg_type]
-
-    if context.line is None:
-        lineno = -1  # type: ignore[unreachable]
-    else:
-        lineno = context.line
-
-    if context.function is None:
-        func = 'none'  # type: ignore[unreachable]
-    elif ':' in context.function:
-        func = '"{}"'.format(context.function)
-    else:
-        func = context.function
-
-    if context.category is None or context.category == 'default':
-        name = 'qt'
-    else:
-        name = 'qt-' + context.category
-    if msg.splitlines()[0] == ('This application failed to start because it '
-                               'could not find or load the Qt platform plugin '
-                               '"xcb".'):
-        # Handle this message specially.
-        msg += ("\n\nOn Archlinux, this should fix the problem:\n"
-                "    pacman -S libxkbcommon-x11")
-        faulthandler.disable()
-
-    assert _args is not None
-    if _args.debug:
-        stack: Optional[str] = ''.join(traceback.format_stack())
-    else:
-        stack = None
-
-    record = qt.makeRecord(name=name, level=level, fn=context.file, lno=lineno,
-                           msg=msg, args=(), exc_info=None, func=func,
-                           sinfo=stack)
-    qt.handle(record)
-
-
 @contextlib.contextmanager
 def hide_qt_warning(pattern: str, logger: str = 'qt') -> Iterator[None]:
```

Comment:
Even if it's not required import-wise, just to keep related things together, this should probably be moved as well.

Reply:
Fully agree, I will move this piece, as well.